src/main/java/io/github/mountainrange/mule/controllers/PlayController.java

```diff
@@ -27,7 +26,7 @@
 
 	private SceneLoader sceneLoader;
 	private MULE mule;
-	private VisualGrid<VisualTile> g;
+	private VisualGrid g;
 	private WorldMap map;
```

Comment:
PlainTile might work? I'm not sure. I don't see why this is a problem, it dosent make sense that the compiler dosen't like it.